### PR TITLE
ADD: Warning for missing tag on Interactables & Billboard Start

### LINKED ISSUE #16: Trees change to billboards and back too quickly
Terrain settings -> Billboard start

### LINKED ISSUE #44: Auto-Assign GameObject Tag for Interactable when script is attached
When attaching an Interactable script, the tag "Interactable" is required to be attached to the gameobject. This can be outsourced to the script.

### LINKED ISSUE #144: Inventory: Hovering over item has no effect

### PR DESCRIPTION
**Pull request by Mimonsi**
closes #16 
closes #44

### CODE REVIEW

File: Assets/Scripts/Interaction/Interactable.cs [hunk 1]
```diff
@@ -13,6 +13,15 @@ public abstract class Interactable : MonoBehaviour
 {
     public string displayText;
     public string displaySubtext;
+    public bool ignoreTagCheck = false;
```
Comment (iTitus) [inline Assets/Scripts/Interaction/Interactable.cs:16]: what does this field do?

**Review by iTitus — CHANGES_REQUESTED** [review]
Why does this close #144 ?

File: Assets/Scripts/Interaction/Interactable.cs (hunk 1 above)
Comment (Mimonsi) [reply Assets/Scripts/Interaction/Interactable.cs:16]: This field can be checked, if you want to ignore the warning for a GameObject in the scene - for whatever reason. It was just easy to implement and might be helpful

**Review by iTitus — APPROVED** [review]
🥇

**Pull request merged**